pkg/util/cmdutil/diag.go

```diff
@@ -67,3 +67,11 @@ func InitDiag(opts diag.FormatOptions) {
 	contract.Assertf(snk == nil, "Cannot initialize diagnostics sink more than once")
 	snk = diag.DefaultSink(os.Stdout, os.Stderr, opts)
 }
+
+// Plural uses a dumb pluralization scheme, simply appending an "s", to the given string s if c is not 1.
+func Plural(s string, c int) string {
+	if c != 1 {
+		s += "s"
+	}
+	return s
+}
```

Comment:
lol.  is tehre no Humanize Pluralize library already for go?

Reply:
Oh, neat, go-humanize does pluralization: https://github.com/dustin/go-humanize. I'll switch to that. It can't be worse than my lame heuristic (which, to be clear, already existed in our code prior to this change 🤣 )